fdbserver/Resolver.actor.cpp

```diff
@@ -18,12 +18,14 @@
  * limitations under the License.
  */
 
+#include <algorithm>
```

Comment:
I don't see where values from this header are used.

Reply:
I added it for std::max. I'm not calling it any more, but there is another call to it in this file.